chanfitness/chanevent.go

```diff
@@ -95,14 +121,34 @@ func (p *peerLog) addOnlineEvent(online bool, time time.Time) {
 	eventType := peerOnlineEvent
 	if !online {
 		eventType = peerOfflineEvent
+		p.flapCount++
```

Comment:
Adding to the prev comments: we could increase flap count for subsequent online events too.

Reply:
One thing we may need to account for if we increment for online events is that we will then do so on restart for every peer (not just those which are offline when we restart). Perhaps this is more fair, because the online peers may have flapped while our node was down, and the offline ones just have bad luck.